src/js/features/progress.js

```diff
@@ -102,17 +118,18 @@ Object.assign(MediaElementPlayer.prototype, {
 
 					pos = x - offset.left;
 					percentage = (pos / width);
-					newTime = (percentage <= 0.02) ? 0 : percentage * media.duration;
+					t.newTime = (percentage <= 0.02) ? 0 : percentage * media.duration;
 
-					// seek to where the mouse is
-					if (mouseIsDown && newTime.toFixed(4) !== media.currentTime.toFixed(4)) {
-						media.setCurrentTime(newTime);
-					}
+					// fake seek to where the mouse is 
+					if (mouseIsDown && t.newTime.toFixed(4) !== media.currentTime.toFixed(4)) { 
+						t.setCurrentRailHandle(t.newTime);
```

Comment:
I see. Well I'll add that option, fix your changes and merge into master. Thanks